include/nbl/ui/CWindowManagerWin32.h

```diff
@@ -20,7 +20,26 @@ namespace nbl::ui
 	public:
 		CWindowManagerWin32() = default;
 		~CWindowManagerWin32() {};
-
+		virtual core::smart_refctd_ptr<IWindow> createWindow(IWindow::SCreationParams&& creationParams) override
+		{
+			CWindowWin32::native_handle_t handle = createNativeWindow(creationParams.x,
+				creationParams.y,
+				creationParams.width,
+				creationParams.height,
+				creationParams.flags,
+				creationParams.windowCaption);
+			// TODO: params validation
+			if (false)
+			{
+				return nullptr;
+			}
+			return core::make_smart_refctd_ptr<CWindowWin32>(this, std::move(creationParams), handle);
+		}
+		virtual void destroyWindow(IWindow* wnd) override
```

Comment:
unless you plan on inheriting from this class again and overriding these overrides, dont put `virtual` in front of the function (moreover you could make this the `final` override)